taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path

```

Comment:
add blank line between different import groups

use absolute import instead of relative